### PR TITLE
[oap-native-sql] Read next vectors directly from input stream intead o…

### PR DESCRIPTION
**Pull request by xuechendi**
…f using ArrowStreamReader

Signed-off-by: Chendi Xue <chendi.xue@intel.com>

Fixed: https://github.com/Intel-bigdata/OAP/issues/1636
Fixed: https://github.com/Intel-bigdata/OAP/issues/1607

### CODE REVIEW

**Comment by xuechendi** [comment]
@zhztheplayer , hongze, could you have a try ? I now read vectors directly from raw instream to avoid ownership issue, I think it should be more straight forward.

**Comment by zhztheplayer** [comment]
OK, I'll take a look by tomorrow, thanks

**Comment by xuechendi** [comment]
@zhouyuan @zhztheplayer 
This PR passed tpch and tpcds in my setup, thanks

**Comment by zhztheplayer** [comment]
Looks good to me, please additionally resolve #1607 when merging.

**Comment by zhouyuan** [comment]
this pr is merged with #1639

File: oap-native-sql/core/src/main/scala/com/intel/oap/expression/ConverterUtils.scala [hunk 1]
```diff
@@ -131,58 +135,94 @@ object ConverterUtils extends Logging {
...
+          return false
+        }
       override def next(): ColumnarBatch = {
-        try {
-          if (input.available == 0) {
-            array_id += 1
-            input = new ByteArrayInputStream(data(array_id))
-            reader = new ArrowStreamReader(input, ArrowWritableColumnVector.getNewAllocator)
-            root = reader.getVectorSchemaRoot()
+        //try {
```
[the remark this answers is not in the document]
Comment (zhouyuan) [reply oap-native-sql/core/src/main/scala/com/intel/oap/expression/ConverterUtils.scala]: @LuciferYang addresse in https://github.com/Intel-bigdata/OAP/pull/1639/commits/b042a8c9bf45f17d08f2152cb0f41381028224f4

**Comment by zhouyuan** [comment]
merged in #1639 already

**Pull request closed without merging**